enhancements/platform-validation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Kepler Platform Validation
+authors:
+  - Jie Ren (jie.ren@intel.com)
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2023-05-27
+last-updated: 2023-05-27
+tracking-links: # link to related GitHub issues
+  - TBD
+---
+
+# Kepler Platform Validation Guide
+
+## Summary
+
+A quick start guide to validate Kepler support on specific hardware platform.
+
+## Motivation
+
+Currently, Kepler's test cases focus on code logic verification, lack of actual platform validation.
+
+Kepler's [data](https://github.com/sustainable-computing-io/kepler/tree/main/data) files majorly contain static information, or lack of data sync mechanism.
+
+Test cases and test report are not available, which could be the evidence or check criteria to below concerns:
+
+1. Whether the specific CPU/GPU/FPGA/Accelerator model is well supported in Kepler?
+
+2. Whether the data/metrics of specific power source are correctly exported?
```

Comment:
very curious on how this can be done ... e.g calculate the pod energy then aggregate them and compare to RAPL or other measurement tool?

Reply:
Something like what you said, Ji.
RAPL is of coz a way to do comparison, and we could also compare the specific delta, both for Dyn and Abs power.
And if other power metering tools data is available, also could be the check base.
So I suppose test plan is general, test case steps could be hardware specific.
Anyway, such test cases' goal is to let customer believe what we Kepler exported metrics are make sense and acceptable for following operations, i.e. model training, carbon footprint, etc.

Reply:
>>>Anyway, such test cases' goal is to let customer believe what we Kepler exported metrics are make sense and acceptable for following operations, i.e. model training, carbon footprint, etc.

right, don't get me wrong :) 
I am in favor of doing this but just curious on how we can do the test and have a stable (or maybe some smarter way)
way to do the calculation

Reply:
Curious on how this can be done. +1, but I am good here as motivation.